vm/factory/systemSCFactory.go

```diff
@@ -120,6 +124,24 @@ func (scf *systemSCFactory) createGasConfig(gasMap map[string]map[string]uint64)
 	return nil
 }
 
+// GasScheduleChange is called when gas schedule is changed, thus all contracts must be updated
+func (scf *systemSCFactory) GasScheduleChange(gasSchedule map[string]map[string]uint64) {
```

Comment:
It is not needed a mutExecution also here?

Reply:
no - this is a factory.